src/diffusers/loaders/lora_pipeline.py

```diff
@@ -4249,7 +4249,32 @@ def lora_state_dict(
 
         return state_dict
 
-    # Copied from diffusers.loaders.lora_pipeline.CogVideoXLoraLoaderMixin.load_lora_weights
+    @classmethod
+    def _maybe_expand_t2v_lora_for_i2v(
+        cls,
+        transformer: torch.nn.Module,
+        state_dict,
+    ):
+        if any(k.startswith("blocks.") for k in state_dict):
+            num_blocks = len({k.split("blocks.")[1].split(".")[0] for k in state_dict})
+            is_i2v_lora = any("k_img" in k for k in state_dict) and any("v_img" in k for k in state_dict)
```

Comment:
```suggestion
            is_i2v_lora = any("add_k_proj" in k for k in state_dict) and any("add_v_proj" in k for k in state_dict)
```
`k_img` and `v_img` are the original keys

Reply:
Not sure I follow, `k_img` and `v_img` are the extra keys we have in I2V LoRAs, so if we already have these keys we just return the state dict 

for example (squish I2v lora, vs flat color t2v lora)
![Screenshot 2025-03-19 at 15 30 20](https://github.com/user-attachments/assets/c161100c-2ab1-4715-8f17-060faa9b92e9)

![Screenshot 2025-03-19 at 15 30 11](https://github.com/user-attachments/assets/512439c4-3d4e-4696-ac03-a5aeb44dfc26)

Reply:
https://github.com/huggingface/diffusers/blob/fc28791fc8f4df6ac74afd1d0a0e0f1cea67aeec/src/diffusers/loaders/lora_conversion_utils.py#L1380-L1386

`k_img` and `v_img` are the non-Diffusers key, they are mapped to Diffusers keys `add_k_proj` and `add_v_proj` in `_convert_non_diffusers_wan_lora_to_diffusers` which happens during `lora_state_dict`

https://github.com/huggingface/diffusers/blob/fc28791fc8f4df6ac74afd1d0a0e0f1cea67aeec/src/diffusers/loaders/lora_pipeline.py#L4241-L4242